src/capi/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+target_sources(libmahjong
+    PRIVATE
+        capi.cc
```

Comment:
Should be .c if it is a C file. Oh but I guess we're currently doing an extern C around the whole file.

Reply:
Parts are referencing c++ namespaces and classes so I think it needs to be a cpp file?? I could be wrong though I can try it